README.md

```diff
@@ -97,17 +97,29 @@ By default, the document will attempt to be decoded as utf8. This is the default
 
 ### configure(options: IOptions)
 
-Configures linez to use the supplied options. Currently, only the newlines property is available, where you can specify any number of newline character sequences.
+Set the globle configures linez to use the supplied options. you can specify any number of newline character sequences in newlines property, and you can specify any number of code block options in block property.
 
 ```js
 linez.configure({
-  newlines: ['\n', '\r\n', '\r', '\u000B']
+  newlines: ['\n', '\r\n', '\r', '\u000B'],
+  blocks: [
+    {
+      type: 'multilineString',
+      start: /[^\\]".*\\\s*$/,
+      end: /[^\\]";?\s*$/,
+    },
+    {
```

Comment:
I realize that we're trying to add support for multi-line comments for ECLint. Maybe it's best the readme stick to the multi-line comment case?